subworkflows/local/eggnog/main.nf

```diff
@@ -21,6 +21,7 @@ workflow EGGNOG {
         .combine(EGGNOG_DOWNLOAD.out.taxa_db)
         .combine(EGGNOG_DOWNLOAD.out.pkl)
         .set{ ch_eggnog_database }
+    ch_versions = ch_versions.mix ( EGGNOG_DOWNLOAD.out.versions )
```

Comment:
Was it missing from before? if so, did you test both eggnog_download and kofamscan_download?

Reply:
No, I didn't test either of them and neither of them output versions so it won't work. I added this because linting complained, but I'll remove this addition.